packages/api/src/services/actions/global.ts

```diff
@@ -1,33 +1,71 @@
 import assert from "assert";
-import { Json, Actions, AppState, CurrencySymbol } from "../..";
+import { Json, Actions, AppState, CurrencySymbol, AllContractStates } from "../..";
 import * as Queries from "../../libs/queries";
 import bluebird from "bluebird";
 import { BigNumber } from "ethers";
+import { nowS } from "../../libs/utils";
 
 // actions use all the app state
 type Dependencies = AppState;
 type Config = undefined;
 
 export function Handlers(config: Config, appState: Dependencies): Actions {
-  const queries = {
-    emp: Queries.Emp(appState),
-    lsp: Queries.Lsp(appState),
-  };
+  const { stats } = appState;
+  const queries = [Queries.Emp(appState), Queries.Lsp(appState)];
 
   const actions: Actions = {
     echo(...args: Json[]) {
       return args;
     },
+    async listActive() {
```

Comment:
these are the new global api calls. talked to francesco about what he needs, and basically its just the getState ( for any contract), and global tvl and global tvl history.  I added listExpired and listActive since this was easy and gives us a way to see all contracts